animations.js

```diff
@@ -1,33 +1,9 @@
-//var el = document.getElementById('calendar')
+let elMenuItem = document.getElementsByClassName("item-link")[0];
 
-//el.onclick = function () {
-//console.log('Click just happened')
-//}
+// First Icon event handling
+elMenuItem.addEventListener('click', () => {
```

Comment:
Now, to have to add an event listener to each one of the menu items, you could hardcode it and copy-paste, or you could DRY it.

Hint: look for ways to do loops in JS. [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) can be a solution.